lib/Channel.js

```diff
@@ -147,8 +148,11 @@ class Channel extends EnhancedEventEmitter
 			}
 		});
 
-		this._socket.on('end', () => this._logger.debug('Channel ended by the worker process'));
-		this._socket.on('error', (error) => this._logger.error('Channel error: %s', String(error)));
+		this._consumerSocket.on('end', () => this._logger.debug('Consumer channel ended by the worker process'));
```

Comment:
Let's log "Consumer Channel ..." and "Producer Channel..." instead (here and below).

Reply:
yep. already changed.